gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -848,8 +848,13 @@ async def test_{{ method_name }}_async_pages():
         {% endif %}
         )
         pages = []
+        # Workaround issue in python 3.9 related to code coverage by adding `# pragma: no branch`
+        # See https://github.com/googleapis/gapic-generator-python/pull/1174#issuecomment-1025132372
+        # Turn off code formatting as `# pragma: no branch` does not work on multilines
```

Comment:
As per https://stackoverflow.com/a/54134770, it may work if you put it before the line break,
so maybe something like this would work?
```
async for page_ in ( #pragma no branch
   await client.{{ method_name }}(request={})).pages:
  pages.append(page_)
```

It would be nice if we could get this working instead, but if it's not quick to do right now, I'm OK filing a follow-up issue and proceeding with this change, since we're disabling formatting rather than coverage.

Reply:
Nice find! This worked.

Reply:
Fixed in a8463aa